web-local/src/lib/application-state-stores/app-store.ts

```diff
@@ -24,6 +25,9 @@ const appStoreReducers = {
         type,
       };
       state.previousActiveEntity = state.activeEntity;
+      if (state.previousActiveEntity) {
```

Comment:
Where are we using these? I think `activeEntity` has been mostly deprecated from the application at this point right?